### ISSUE TITLE
remove: needs ability to be remove by rowIndex

### ISSUE THREAD

**Issue by Scnck** [wenzhixin/bootstrap-table#4907]
```
	    key: "remove",
	    value: function remove(params) {
	      var len = this.options.data.length;
	      var i;
	      var row;

	      if (!params.hasOwnProperty('field') || !params.hasOwnProperty('values')) {
	        return;
	      }

		  **if (params.field.includes('rowIndex'))
			  params.values.forEach(rowIndex => this.options.data.splice(rowIndex, 1));**
		  else
	      for (i = len - 1; i >= 0; i--) {
	        row = this.options.data[i];

	        if (!row.hasOwnProperty(params.field)) {
	          continue;
	        }

	        if (params.values.includes(row[params.field])) {
	          this.options.data.splice(i, 1);

	          if (this.options.sidePagination === 'server') {
	            this.options.totalRows -= 1;
	          }
	        }
	      }

	      if (len === this.options.data.length) {
	        return;
	      }

	      this.initSearch();
	      this.initPagination();
	      this.initSort();
	      this.initBody(true);
	    }
```

**Comment by wenzhixin** [comment]
ref #564

**Comment by wenzhixin** [comment]
Fixed in #4909
